template.md

```diff
@@ -0,0 +1,29 @@
+# Summary
```

Comment:
One of the biggest points in Rust's RFC process is "How Will We Teach This?"

You can see some reasoning here: https://github.com/rust-lang/rfcs/blob/master/text/1636-document_all_features.md#new-rfc-section-how-do-we-teach-this

I think it is useful for features since many features we add are adding new functionality that not everyone will understand.

Reply:
@Hoverbear 

I think we can keep thing simple at first and the summary is enough.

Btw, I don't think we need to add like "how to teach this", this RFC is mainly for the developer and mostly I think they don't need to be taught.

Reply:
> Features which affect users (such as introducing joint consensus compared to single add/remove) need to be taught to the user effectively.

For this, I think we need to explain in the "Detailed Design" section. Here is only a summary.

Reply:
Okay. :)